src/dev/src/app/timeline/timeline.demo.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+
+import { ClarityIcons } from '@clr/icons';
+
+@Component({
+  selector: 'clr-timeline-demo',
+  styleUrls: ['./timeline.demo.scss'],
+  template: `
+        <h2>Timelines</h2>
+        <ul>
+            <li><a [routerLink]="['./timeline-types']">Timeline types</a></li>
+        </ul>
+        <router-outlet></router-outlet>
+    `,
+})
+export class TimelineDemo {
+  ngOnInit() {
+    ClarityIcons.add({
+      spinner: `<svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36"><title>spinner 23</title><rect width="36" height="36" fill-opacity="0"/><circle cx="18" cy="18" r="14.85" fill="none" stroke="#000" stroke-miterlimit="10" stroke-opacity="0.15" stroke-width="2.3"/><path d="M8.24,29.2A14.86,14.86,0,0,1,18,3.15" fill="none" stroke="#007cbb" stroke-miterlimit="10" stroke-width="2.3"/></svg>`,
```

Comment:
For this to work, it seems we need to have a spinner icon, and not have it hard coded here. Can we add this to our icon core library? @Shijir does that make sense?

Reply:
@Shijir Though in our previous conversation we discussed that the icon could be added using `ClarityIconApi` by our own, we still need it to be part of Clarity official icon so that consumers using static version of timeline can see the icon. Is that correct?

Reply:
This icon was supposed to be added into the module of this component. Client apps that don't have access to this icon won't be able to render this icon.

Reply:
Ok we'll leave this for now, but need to figure this out before we announce this functionality.